wordpress-mobile/WordPress-Android
# Update donut chart label formatting

Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/19959


This PR updates the formatting for the donut chart legend. 
| Before | After |
|--------|-------|
|    <img width="285" alt="Screenshot 2024-04-18 at 3 53 07 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/a15cbeed-6688-4d36-8c00-8c69c32dece6">    |  <img width="291" alt="Screenshot 2024-04-18 at 3 53 17 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/962f5588-22fd-4698-ac96-c70068ca6d4e">  |
|     <img width="288" alt="Screenshot 2024-04-18 at 3 52 09 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/70b3149a-24d4-4e40-9053-2da516831062">  |   <img width="285" alt="Screenshot 2024-04-18 at 3 52 21 PM" src="https://github.com/wordpress-mobile/WordPress-Android/assets/7199140/aca5a816-c891-4aa3-8cb4-212aba64f0ef">  |

-----

## To Test:
1. Navigate to Stats -> Insights -> Total Followers. 
2. Ensure that the legend is formatted well and test with different device font sizes.


-----

## Regression Notes

1. Potential unintended areas of impact

    - None

3. What I did to test those areas of impact (or what existing automated tests I relied on)

    - None

4. What automated tests I added (or what prevented me from doing so)

    - None

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
